pyvista/plotting/themes.py

```diff
@@ -1469,6 +1479,33 @@ def server_proxy_prefix(self):  # numpydoc ignore=RT01
     def server_proxy_prefix(self, prefix: str):  # numpydoc ignore=GL08
         self._server_proxy_prefix = prefix
 
+    @property
+    def jupyter_extension_available(self) -> bool:  # numpydoc ignore=RT01
+        """Return whether the trame_jupyter_extension is detected."""
+        return self._jupyter_extension_available
+
+    @jupyter_extension_available.setter
+    def jupyter_extension_available(self, _available: bool):  # numpydoc ignore=GL08
+        warnings.warn(
+            "The jupyter_extension_available flag is read only and is automatically detected."
+        )
+
```

Comment:
Originally I omitted the setter like you suggest here, however one of the unit tests was failing because the setter was missing, and that's why I ended up adding a noop setter.

Agreed that the theme might not be the best place for available flag. If you have a better place in mind, I'll be happy to move it there

Reply:
Ah, I didn't realize a test was complaining about this. I think let's just leave this as is then. Thanks for looking into this!